data_write/data_write.py

```diff
@@ -327,10 +320,36 @@ def parse_antenna_iq(self):
                         antenna_iq_stage[ant_str] = {}
 
                     if 'data' not in antenna_iq_stage[ant_str]:
-                        antenna_iq_stage[ant_str]['data'] = antennas_data[ant_num, :]
-                    else:
-                        arr = antenna_iq_stage[ant_str]
-                        arr['data'] = np.concatenate((arr['data'], antennas_data[ant_num, :]))
+                        antenna_iq_stage[ant_str]['data'] = []
+                    antenna_iq_stage[ant_str]['data'].append(antennas_data[ant_num, :])
+
+    def numpify_arrays(self):
```

Comment:
I updated this, now it gets called at the end of an averaging period before data is written to file. Also did some rearrangements to how data is parsed, so that shared memory isn't kept around as long.